src/components/FilmCard.vue

```diff
@@ -52,6 +52,9 @@ export default {
     onCardClick() {
       console.log("FilmCard#onCardClick");
       this.$emit(EVENTS.FILM_CARD_CLICKED);
+    },
+    setDefaultImage(event) {
+      event.target.src = require(`@/assets/300x450.png`);
```

Comment:
This string should be in "core/constants"

Reply:
I managed to do it only this way

> const { EVENTS, APP_CONFIG } = require("../config/constants");

> event.target.src = require(`@/${APP_CONFIG.DEFAULT_IMAGE_PATH}`);

In other cases it fails with `@/assets/300x450.png not found` or `APP_CONFIG is not defined`